backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/codec/impl/AbstractElasticsearchJavaTimeFieldCodec.java

```diff
@@ -31,6 +31,15 @@ public JsonElement encode(T value) {
 		return new JsonPrimitive( nullUnsafeFormat( value ) );
 	}
 
+	@Override
+	public JsonElement encodeForMissing(T value) {
```

Comment:
Oh, and maybe it would be a good idea to add a comment here, referencing HSEARCH-3255 and explaining why we need this? It will look odd to both of us in a few months...